src/dispatch/static/dispatch/src/plugin/store.js

```diff
@@ -83,7 +83,8 @@ const actions = {
             "app/SET_SNACKBAR",
             {
               text: "Plugin not created. Reason: " + err.response.data.detail,
-              color: "red"
+              color: "red",
+              timeout: 0
```

Comment:
If we're happy with this, I can go and add it to the rest of stores.

Reply:
If we want to force users to always close `error` messages (instead of timing out). I wonder if we should instead push that down into the component itself. 

i.e. we would pass a `type` prop to to the Snackbar component. When the component encounters any "error" type messages it will set the timeout to `0` and the ` color to `red`.

We can then also define `success` and `info` type messages. This way, if we want to change how those messages behave/look we only need to deal with the component (instead of modifying all messages like we have to now).